applications/app/services/ApplicationsSpecial2020Election.scala

```diff
@@ -25,18 +37,12 @@ object ApplicationsSpecial2020Election {
   }
 
   def ampTagHtml(path: String)(implicit request: RequestHeader): Html = {
-    ApplicationsDotcomRenderingInterface.getRenderingTier(path) match {
-      case Election2020Hack => {
-        Html(
-          // The following simple string interpolation wasn't compiling.
-          // (probable reason: Proximity of string interpolation and quote escaping.)
-          // s"<link rel=\"amphtml\" href=\"https://amp.theguardian.com/${path}\">",
-
-          // Going for another solution then
-          Array("<link rel=\"amphtml\" href=\"https://amp.theguardian.com/", path, "\">").mkString(""),
-        )
-      }
-      case _ => Html("")
+    if (ApplicationsSpecial2020Election.pathIsSpecialHanding(path)) {
+      Html(
+        s"""<link rel="amphtml" href="https://amp.theguardian.com/${path}">""",
```

Comment:
This won't be double slashed now will it? `/${path}` path being `"/worl...`?

Reply:
Nope. I accidentally pointed that out in the comment :)
```
when called from `ApplicationsDotcomRenderingInterface.getRenderingTier` it comes without starting slash
```

Reply:
Oh! Jezz. Yes. Let me correct that.